WalletWasabi/Models/SmartCoin.cs

```diff
@@ -99,19 +99,11 @@ public Height Height
 		/// Always set it before the Amount!
 		/// </summary>
 		[JsonProperty]
-		public string Label
+		[JsonConverter(typeof(LabelJsonConverter))]
```

Comment:
```suggestion
		[JsonConverter(typeof(SmartLabelJsonConverter))]
```

Reply:
Thanks, I did it manually instead of accepting your commit suggestions, to make sure I won't miss out anything.